DiamondLightSource/dodal
# 3: Add system tests to check for connection

Fixes #3 

Note, branched off #7 so merge that first

To test:
* Run system tests `pytest tests/test_beamlines_connect.py` and confirm they pass for i03 (s03 is not currently running)
